README.md

```diff
@@ -1,21 +1,147 @@
-A template repository to make it easy to create new prismarine repo
+Node.JS Wrapper around both the Java & Bedrock Realms API. Minecraft Realms is a subscription based service provided by Mojang where users can create/manage their own private server. The intention of this package is to provide easy access to the internal API used to manage Realms such as opening/closing a Realm, managing players, getting host information and much more. 
+
+## Installation
+```shell
+npm install prismarine-realms
+```
 
 ## Usage
 
+### RealmAPI
+
+#### .from(authflow: Authflow, platform: 'bedrock' | 'java')
+
+Takes an **Authflow** instance from [prismarine-auth](https://github.com/PrismarineJS/prismarine-auth), you can see the documentation for this [here.](https://github.com/PrismarineJS/prismarine-auth#authflow)
+
+## API
+
+### Definitions
+
+| Param           | Type                 | Description                                                           |
+| --------------- | -------------------- | --------------------------------------------------------------------- |
+| realmId         | `string`             | The ID of the Realm                                                   |
+| realmInviteCode | `string`             | The invite code for the Realm (Only on Bedrock)                       |
+| username        | `string`             | The username of player                                                |
+| uuid            | `string`             | The unique ID of the player, w/o hyphens                              |
+| xuid            | `string`             | The Xbox User ID of the targeted player                               |
+
+---
+
 ```js
-const template = require('prismarine-template')
+const { Authflow } = require('prismarine-auth') 
+const { RealmAPI } = require('prismarine-realms')
 
-template.helloWorld()
+const authflow = new Authflow()
+
+const api = RealmAPI.from(authflow, 'bedrock')
 ```
 
-## API
+#### getRealms()
+
+Gets a list of Realms the authenticating account has joined or owns
+
+```js
+await api.getRealms()
+```
 
-### helloWorld()
 
-Prints hello world
+#### getRealm(realmId: string)
+
+Gets detailed information about a Realm if owned
+
+```js
+await api.getRealm('1234567')
+```
+
+#### getAddress()
+
+Gets the address for the Realm
+
+```js
+const realm = await api.getRealm('1234567')
+
+await realm.getAddress()
+```
+
+#### invitePlayer(uuid: string, name: string)
+
+Invites a player to the Realm
+
+```js
+const realm = await api.getRealm('1234567')
+
+await realm.invitePlayer('a8005260a332457097a50bdbe48a9a21', 'Steve')
+```
+
+#### open()
+
+Opens a Realm. Allows players to join
+
+```js
+const realm = await api.getRealm('1234567')
+
+await realm.open()
+```
+
+#### close()
+
+Closes a Realm. Removes all current players and restricts joining
+
+```js
+const realm = await api.getRealm('1234567')
+
+await realm.close()
+```
+
+---
+
+### Using prismarine-realms with Mineflayer, Node Minecraft Protocol and Bedrock Protocol
```

Comment:
Are you happy with the wording here @extremeheat and the examples in terms of how we will integrate Realms?

Reply:
OK, but I think this should only include information that is actually true right now. Once it’s implemented in other places we can add this explanation here, but I feel this bit belongs more inside the protocol lib than it does here as an independent package

Reply:
Yeah maybe would be a better place to put it. The change was to resolve Roms review https://github.com/PrismarineJS/prismarine-realms/pull/1#discussion_r825518646